arcs/src/algorithms/scale_non_uniform.rs

```diff
@@ -1,79 +1,51 @@
 use crate::{
-    primitives::{Point, Line},
     Vector,
+    algorithms::{AffineTransformable},
 };
 use kurbo::Affine;
 
 /// Something which can be scaled **non-uniform** in x and y directions in *Drawing Space*
 pub trait ScaleNonUniform {
-    fn scale_nu(&mut self, factor_x: f64, factor_y: f64, base: Vector);
+    fn scale_non_uniform(&mut self, factor_x: f64, factor_y: f64, base: Vector);
 
-    fn scaled_nu(&self, factor_x: f64, factor_y: f64, base: Vector) -> Self
+    fn scaled_non_uniform(&self, factor_x: f64, factor_y: f64, base: Vector) -> Self
     where 
         Self: Sized + Clone,
         {
             let mut clone = self.clone();
-            clone.scale_nu(factor_x, factor_y, base);
+            clone.scale_non_uniform(factor_x, factor_y, base);
 
             clone
         }
 
 }
 
-impl<'t, T: ScaleNonUniform + ?Sized> ScaleNonUniform for &'t mut T {
-    fn scale_nu(&mut self, factor_x: f64, factor_y: f64, base: Vector) {
-        (*self).scale_nu(factor_x, factor_y, base);
+impl<A: AffineTransformable> ScaleNonUniform for A {
+    fn scale_non_uniform(&mut self, factor_x: f64, factor_y: f64, base: Vector){
+        // TODO: Change to `Affine::scale_non_uniform()` after crates.io update
```

Comment:
After a `kurbo` version update we can change this to `Affine::scale_non_uniform(factor_x, factor_y)`